src/tests/ftest/util/dmg_utils.py

```diff
@@ -433,7 +433,7 @@ def pool_create(self, scm_size, uid=None, gid=None, nvme_size=None,
         else:
             match = re.findall(
                 r"UUID:\s+([A-Za-z0-9-]+),\s+"
-                r"Service replicas:\s+([A-Za-z0-9-]+)",
+                r"Service replicas:\s+([0-9]+(,[0-9]+)*)",
```

Comment:
To capture the entire list, e.g. "1,2,3", you'll want to use this pattern:
```suggestion
                r"Service replicas:\s+([A-Za-z0-9-,]+)",
```